src/hooks/use-archive-site.ts

```diff
@@ -97,9 +94,13 @@ export function useArchiveSite() {
 					return;
 				}
 
-				const file = new File( [ archiveContent ], 'loca-env-site-1.zip', {
-					type: 'application/zip',
-				} );
+				const file = new File(
+					[ await getIpcApi().getFileContent( archivePath ) ],
+					'loca-env-site-1.zip',
```

Comment:
What's the story behind this file name here? Didn't try to fix what seems like a typo

Reply:
Yeah, it's a typo. Studio was called "Local Environment" before it had name, so that's the reason behind the archive name.
I thought we could use the site names here, but my intention was to display that we could use any random name here. Maybe we could update it to studio-site.zip